phoenix-core/src/test/java/org/apache/phoenix/mapreduce/index/IndexScrutinyTableOutputTest.java

```diff
@@ -43,7 +43,7 @@ public void testConstructMetadataParamQuery() {
     @Test
     public void testGetSqlQueryAllInvalidRows() throws SQLException {
         SourceTargetColumnNames columnNames =
-                new SourceTargetColumnNames.DataSourceColNames(pDataTable, pIndexTable);
+                new SourceTargetColumnNames.DataSourceColNames(pDataTable, pIndexTable, false);
```

Comment:
Having a new constructor that takes the extra boolean parameter would eliminate changing the existing code.

Reply:
ok